### PR TITLE
Fix build process

### PR DESCRIPTION
**Pull request by jeppes**
Remove all the annoying parts of the build systems, remove half the dependencies, and fix all linter errors. Update to react 16, react router 4.

### CODE REVIEW

**Comment by jeppes** [comment]
@studieresan/studs2018 There is probably too much code to go through, so just doing some testing might suffice

File: app/App.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          <App>
+            <StudsRouter/>
+          </App>
+        </ConnectedRouter>
+      </LanguageProvider>
+    </Provider>
+  }
+}
+
+// Chunked polyfill for browsers without Intl support
```
Comment (jeppes) [inline app/App.js]: will need to deal with this at some point

File: .editorconfig [hunk 2]
```diff
@@ -1,7 +0,0 @@
-root = true
-
-[*]
-end_of_line = lf
-insert_final_newline = true
-indent_style = space
-indent_size = 2
```
Comment (aronstrandberg) [inline .editorconfig]: Why remove this?

File: app/components/HomePage/HomePageHeader/styles.css [hunk 3]
```diff
@@ -4,7 +4,7 @@
   display: flex;
   justify-content: center;
   align-content: center;
-  background: linear-gradient(to right, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('static/img/flower-pattern-1.svg');
+  background: linear-gradient(to right, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('../../../static/img/flower-pattern-1.svg');
```
Comment (aronstrandberg) [inline app/components/HomePage/HomePageHeader/styles.css]: I would prefer to keep this as an absolute import.

File: app/components/HomePage/HomePageSalesContact/styles.css [hunk 4]
```diff
@@ -3,7 +3,7 @@
   align-items: center;
   flex-direction: column;
   background-color: #072C2E;
-  background-image: linear-gradient(to right, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('static/img/flower-pattern-1.svg');
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('../../../static/img/flower-pattern-1.svg');
```
Comment (aronstrandberg) [inline app/components/HomePage/HomePageSalesContact/styles.css]: Absolute path here as well

File: app/containers/HomePage/index.js [hunk 5]
```diff
@@ -19,20 +8,20 @@ import {
   HomePageHeader,
   HomePagePitch,
   HomePageSalesContact,
-} from 'components/HomePage'
+} from '../../components/HomePage'
```
Comment (aronstrandberg) [inline app/containers/HomePage/index.js]: Absolute path here as well

File: app/containers/NotFoundPage/index.js [hunk 6]
```diff
@@ -1,25 +1,20 @@
...
- * component (SFC), hot reloading does not currently support SFCs. If hot
- * reloading is not a necessity for you then you can refactor it and remove
- * the linting exception.
- */
-
 import React from 'react'
 import { FormattedMessage } from 'react-intl'
+import { Link } from 'react-router-dom'
 import messages from './messages'
+import styles from './style.css'
```
Comment (aronstrandberg) [inline app/containers/NotFoundPage/index.js]: Rename this to `styles.css` to match other files in the project?

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Studs 18</title>
```
Comment (aronstrandberg) [inline public/index.html]: I would prefer to keep this as `Studs 2018` as it is currently

**Comment by jeppes** [comment]
@aronstrandberg good points, thanks. Fixed them.

**Pull request merged**